### PR TITLE
Remove mzroll export

### PR DESCRIPTION
**Pull request by saifulbkhan**
As intimated, we are slowly moving away from the `mzroll` format. This patch removes the ability to export `mzroll` files from CLI and GUI executables of El-MAVEN.

### CODE REVIEW

File: src/cli/peakdetector/peakdetectorcli.cpp [hunk 1]
```diff
@@ -1135,22 +1119,6 @@ bool PeakDetectorCLI::_sendUserEmail(QMap<QString, QString> creds,
...
 }
 
-void PeakDetectorCLI::saveMzRoll(string setName)
-{
-    if (saveMzrollFile == true) {
-#ifndef __APPLE__
-        double startSavingMzroll = getTime();
-#endif
-
-        writePeakTableXML(setName + ".mzroll");
```
Comment (shubhra-agrawal) [inline src/cli/peakdetector/peakdetectorcli.cpp]: This function is not being used anywhere. Please remove it.

Comment (shubhra-agrawal) [reply src/cli/peakdetector/peakdetectorcli.cpp]: Same for `writeSampleListXML`, `writeParametersXML` and `writeGroupXML`

Comment (saifulbkhan) [reply src/cli/peakdetector/peakdetectorcli.cpp]: Sorry. Missed this. Removed now.

**Review by shubhra-agrawal — APPROVED** [review]

**Comment by shubhra-agrawal** [comment]
Have not tested it but the code looks fine. I'll merge it as soon as the CI is done.

**Pull request merged**